test/runtests.jl

```diff
@@ -91,7 +93,9 @@ try
     NEW = GZip.GZLIB_VERSION > "1.2.3.9"
     pos = position(gzfile)
     NEW && (pos2 = position(gzfile,true))
-    @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
+    try 
+        @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
```

Comment:
When commenting out 'catch' line 96 and replacing 98 with 'println("l98") the result is:
```
julia> include("runtests.jl")
ERROR: LoadError: test failed: seek(gzfile,100) did not throw ErrorException
 in expression: seek(gzfile,100)
 in error at error.jl:21
while loading C:\Users\F\Documents\GitHub\GZip.jl\test\runtests.jl, in expression starting on line 97
```
This in Julia 0.4.6, Windows 10.